fetch_firmware.sh

```diff
@@ -40,31 +40,26 @@ fetch() {
 }
 
 download_fw() {
-  HW_CONFIG=$1
-  FIRMWARE_DIR=firmware/$HW_CONFIG
+  FIRMWARE_DIR=firmware/prod
 
   # Make firmware download dir
   mkdir -p $FIRMWARE_DIR
 
   # Download piksi_firmware
-  fetch $FW_S3_PATH/piksi_firmware_v3_$HW_CONFIG.stripped.elf \
+  fetch $FW_S3_PATH/piksi_firmware_v3_prod.stripped.elf \
     $FIRMWARE_DIR/piksi_firmware.elf
 
   # Download piksi_fpga
-  if [ "$HW_CONFIG" == "microzed" ]; then
-    # Microzed FPGA image breaks the naming convention so deal with it as a special case
-    fetch $NAP_S3_PATH/piksi_microzed_nt1065_fpga.bit $FIRMWARE_DIR/piksi_fpga.bit
-  else
-    fetch $NAP_S3_PATH/piksi_${HW_CONFIG}_fpga.bit $FIRMWARE_DIR/piksi_fpga.bit
-  fi
+  fetch $NAP_S3_PATH/piksi_sdk_fpga.bit $FIRMWARE_DIR/piksi_sdk_fpga.bit
+  fetch $NAP_S3_PATH/piksi_prod_fpga.bit $FIRMWARE_DIR/piksi_prod_fpga.bit
 
 }
 
 if [[ -z "$GENERATE_REQUIREMENTS" ]]; then
-  download_fw "prod" || echo "ERROR: failed to download FPGA and RTOS artifacts"
+  download_fw || echo "ERROR: failed to download FPGA and RTOS artifacts"
 else
   REQUIREMENTS_M4="$D/requirements.yaml.m4"
   REQUIREMENTS_OUT="${REQUIREMENTS_M4%.m4}"
   [[ -f "$REQUIREMENTS_M4" ]] || { echo "ERROR: could not find $REQUIREMENTS_M4"; exit 1; }
   m4 -DFW_VERSION=$FW_VERSION -DNAP_VERSION=$NAP_VERSION $REQUIREMENTS_M4 >$REQUIREMENTS_OUT
-fi
\ No newline at end of file
+fi
```

Comment:
Does it make sense to commit the fetch_firmware.sh generated by my change of fetch_firmware.sh.m4?  `make firmware` is broken without this and `make image*` is limited in which builds are possible

Reply:
Yes, definitely makes sense